app/models/track.rb

```diff
@@ -41,6 +41,10 @@ def git
     Git::Track.new(synced_to_git_sha, repo_url: repo_url)
   end
 
+  def repo
```

Comment:
```suggestion
  def repo_name
```

I get confused about whether `repo` is the repo object (which it is in some places), a URL (https://githbub...) or the name (this?). I think we should try and always suffix the url and the name.

Reply:
Good idea